assets/controller.yaml

```diff
@@ -98,28 +98,6 @@ spec:
             requests:
               memory: 50Mi
               cpu: 10m
-          # kube-rbac-proxy for csi-driver container.
-          # Provides https proxy for http-based csi-driver metrics.
-        - name: driver-kube-rbac-proxy
-          args:
-          - --secure-listen-address=0.0.0.0:9216
-          - --upstream=http://127.0.0.1:8216/
-          - --tls-cert-file=/etc/tls/private/tls.crt
-          - --tls-private-key-file=/etc/tls/private/tls.key
-          - --logtostderr=true
-          image: ${KUBE_RBAC_PROXY_IMAGE}
-          imagePullPolicy: IfNotPresent
-          ports:
-          - containerPort: 9216
-            name: driver-m
-            protocol: TCP
-          resources:
-            requests:
-              memory: 20Mi
-              cpu: 10m
-          volumeMounts:
-          - mountPath: /etc/tls/private
-            name: metrics-serving-cert
```

Comment:
I think you'll also want to remove the associated volume:
https://github.com/openshift/aws-efs-csi-driver-operator/blob/4310edab61d19bfb9245fed2773d834ff5395b34/assets/controller.yaml#L182-L184

Reply:
And where does the `aws-efs-csi-driver-controller-metrics-serving-cert` secret get created? Does the secret also need to be removed?

Reply:
> I think you'll also want to remove the associated volume:

That volume is still used in `provisioner-kube-rbac-proxy` container. I added it back there.

> And where does the aws-efs-csi-driver-controller-metrics-serving-cert secret get created? 

It's created by OCP as result of annotation `service.beta.openshift.io/serving-cert-secret-name: aws-efs-csi-driver-controller-metrics-serving-cert` in `service.yaml` - _something_ generates TLS cert and creates a Secret for us.